index.html

```diff
@@ -34,6 +35,159 @@ <h2>I am a Software Developer</h2>
       <li><a href="#"><i class="fa-brands fa-medium"></i></a></li>
     </ul>
   </section>
-  
+
+  <section class="works-section">
+    <h1>My Recent Works</h1>
```

Comment:
- [ ] Please, consider that your page could have only one `h1` header. All other headers should be `hx` (h2, h3, etc) - please, consider it for the whole page 👍

Reply:
Hi @elmar8287 

Thanks for a detailed and kind review.

Concerning having a single h1 header on a page, can you take a look at the screenshot below from [this article](http://diveintohtml5.it/semantics.html) and let me know what you think?

![image](https://user-images.githubusercontent.com/20554583/161499187-a9313b06-20de-4eae-a3d9-e01ba49f3d93.png)

Thank you.